crates/dtrace_probe/Cargo.toml

```
@@ -7,5 +7,6 @@ edition = "2021"
 color-eyre = "0.6"
 clap = { version = "4.2", features = ["derive"] }
 composer_api = { path = "../composer_api" }
+ctrlc = "3.2"
```

I suppose we're using this library so that we can clean-up properly on exit by calling `dtrace.stop()`, right? Then we might consider this
```suggestion
ctrlc = { version = "3.2", features = ["termination"] }
```
which makes ctrlc handle the SIGTERM and SIGHUP signas as well.

---

A good trick. Thank you :)